### PR TITLE
Update secrets doc

### PR DESCRIPTION
**Pull request by wojtek-t**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2372)

### CODE REVIEW

File: docs/user-guide/secrets/index.md [hunk 1]
```diff
@@ -370,7 +370,10 @@ files.
 **Mounted Secrets are updated automatically**
 
 When a secret being already consumed in a volume is updated, projected keys are eventually updated as well.
-The update time depends on the kubelet syncing period.
+Kubelet is checking whether the mounted secret is fresh on every periodic sync.
+However, it is using its local ttl-based cache for getting the current value of the secret.
+As a result, the total delay from the moment when the secret is update to the moment when new keys are
```
Comment (chenopis) [inline docs/user-guide/secrets/index.md]: small grammar nit pick: should be "...moment the secret is update**d**..."

Comment (wojtek-t) [inline docs/user-guide/secrets/index.md]: Fixed.

**Comment by wojtek-t** [comment]
@yujuhong - PTAL

**Pull request merged**